proposals/notary + tuf.adoc

```diff
@@ -0,0 +1,84 @@
+== Notary & TUF Proposal
+
+*Name of project:* Notary & TUF
+
+*Description:*
+
+The Update Framework (TUF) is a specification designed to solve specifically provenance and trust problems as part of a larger distribution framework.
+
+Notary is a content signing framework implementing the TUF specification in the Go language. The project provides both a client, and a pair of server applications to host signed metadata and perform limited online signing functions. It is the de facto image signing framework in use by Docker, Quay, VMWare, and others.
```

Comment:
> The integration of notary is used to map a human readable name to a sha256 digest in a secure and verifiable way.

Right, but I think you're missing my point. Other people *have* tried to include alternative proposals (Red Hat comes immediately to mind) and they haven't been accepted for a variety of reasons. The only supported way of cryptographically signing image identities in Docker is with Notary, and thus anyone who wants to support secure registries must use Notary.

Whether you could *in theory* add other implementations is not relevant to the discussion of whether the statement

> It is the de facto image signing framework in use by Docker, Quay, VMWare, and others

Is a testament to the popularity of Notary, or a testament to the necessity to use it due to the popularity of Docker. This is yet another reason why I asked for examples of Notary use outside of Docker.